src/main/java/org/apache/maven/plugins/clean/Cleaner.java

```diff
@@ -297,6 +299,12 @@ else if ( file.exists() )
         return result;
     }
 
+    private boolean isSymbolicLink( Path path ) throws IOException
+    {
+        BasicFileAttributes attrs = Files.readAttributes( path, BasicFileAttributes.class, LinkOption.NOFOLLOW_LINKS );
+        return attrs.isSymbolicLink() || ( attrs.isDirectory() && attrs.isOther() );
```

Comment:
Can you add a comment here  that the right hand part represents a junction?

Reply:
I'm experimenting with something like:
```
        Path testDir = Paths.get( "target/test-dir" ).toAbsolutePath();
        Files.createDirectories( testDir );
        Path orgDir = testDir.resolve( "org-dir" );
        Path jctDir = testDir.resolve( "jct-dir" );
        Files.createDirectories( orgDir );

        Process process = new ProcessBuilder()
                .directory( testDir.toFile() )
                .command( "mklink", "/J", jctDir.toString(), orgDir.toString() )
                .start();
        int result = process.waitFor();
```
The idea comes from https://www.geeksforgeeks.org/creating-junction-points/